swf/src/read.rs

```diff
@@ -85,9 +85,7 @@ pub fn read_swf_header<'a, R: Read + 'a>(mut input: R) -> Result<SwfStream<'a>>
     let compression = Reader::read_compression_type(&mut input)?;
     let version = input.read_u8()?;
 
-    // Uncompressed length includes the 4-byte header and 4-byte uncompressed length itself,
-    // subtract it here.
-    let uncompressed_length = input.read_u32::<LittleEndian>()? - 8;
+    let uncompressed_length = input.read_u32::<LittleEndian>()?;
```

Comment:
Note that I'm actually changing the semantics of SwfStream here so that the uncompressed_length that gets returned INCLUDES the initial eight bytes of the header. Previously the semantics was that it excluded those eight bytes, but included the (compressed) remainder of the header.